scripts/results/arrange-data.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016-2019, Regents of the University of Arizona.
+# Author:Wenkai Zheng<wenkaizheng@email.arizona.edu>
+# DESCRIPTION:
+# ------------
+# This file is used for arranging data and print out plot
+# to run it you can have python arrange-data.py parameter(such as jitter, rebuffers or rtt)
+# for stdin you have 3 options
+# 1. no input: just left blank
+# 2. one date as input: month/day/year hour:minute:second for example Nov/4/2019 18:55:59
+# 3. two date as input: month/day/year hour:minute:second - month/day/year hour:minute:second for example Nov/4/2019 18:55:59 - Dec/3/2019 18:55:59 

```

Comment:
Follow the same header format as other scripts: 
https://github.com/chavoosh/ndn-mongo-fileserver/blob/master/scripts/results/session-avg-jitter.py

Start new sentences with a capital letter.
Description does not talk about program usage, it is about what the current program does (what we expect it).
You need to add help script that prints program usage and its options when either the input parameters are incorrect or user passes `-h` or `--help` option. Use `argparse` (https://docs.python.org/3/library/argparse.html).